src/App.js

```diff
@@ -1,14 +1,32 @@
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import Quotes from './components/Quotes';
 import './index.css';
+import Home from './components/Home';
 
 function App() {
   return (
-    <div>
-      <Calculator />
-      <Quotes />
-
-    </div>
+    <>
+      <nav>
+        <h1 className="page-title">Math Magicians</h1>
+        <ul className="nav-list">
+          <li>
+            <Link className="link" to="/">Home</Link>
+          </li>
+          <li>
+            <Link className="link" to="/Calculator">Calculator</Link>
+          </li>
+          <li>
+            <Link className="link q-link" to="/Quotes">Quotes</Link>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- Optional
I would strongly recommend moving the navbar to its own component.

Reply:
Thanks for the Suggestion, Will do!